### PR TITLE
clean view creation and fix #content

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/mediatheque/GalleryCollectionView.coffee [hunk 1]
```diff
@@ -27,12 +27,6 @@ GalleryCollectionView = OrchestraView.extend(
     for mediaKey of @options.medias.get(@options.medias.get('collection_name'))
       @addElementToView (@options.medias.get(@options.medias.get('collection_name'))[mediaKey])
     $(".figure").width @options.domContainer.find("img").width()
-    $(".figure").mouseenter(->
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/mediatheque/GalleryCollectionView.coffee]: Why this part is removed ?

File: BackofficeBundle/Resources/public/js/backboneRouter.js [hunk 2]
```diff
@@ -180,20 +176,6 @@ var OrchestraBORouter = Backbone.Router.extend({
     }); 
   },
 
-  setCurrentMainView: function(view)
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/js/backboneRouter.js]: You can also remove the early following code so : 

      // Contains currentMainView, usefull to unbind event when view is changed
      currentMainView: null,

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/js/backboneRouter.js]: done @itkg-ngilain

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**